library/Fum2GitHub/Util.hs

```diff
@@ -0,0 +1,13 @@
+module Fum2GitHub.Util (
```

Comment:
Or alternatively `Fum2Github.Types` which seems to be common for defining newtypes etc. For example: [`Data.Aeson.Types`](https://hackage.haskell.org/package/aeson-0.8.1.1/docs/Data-Aeson-Types.html)